### PR TITLE
Add git/clone command

### PR DESCRIPTION
**Pull request by devanych**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: src/App/Command/Git/CloneCommand.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+use Symfony\Component\Process\Process;
+use Yiisoft\YiiDevTool\App\Component\Console\PackageCommand;
+use Yiisoft\YiiDevTool\App\Component\Package\Package;
+use Yiisoft\YiiDevTool\App\PackageService;
+
+final class CloneCommand extends PackageCommand
+{
+    protected static $defaultName = 'git/clone';
+    protected static $defaultDescription = 'Clone packages repositories';
```
Comment (vjik) [inline src/App/Command/Git/CloneCommand.php:15]:
```suggestion
    protected static $defaultDescription = 'Clone package repositories';
```

**Review by vjik — CHANGES_REQUESTED** [review]
Need add info about command to readme

File: src/App/Command/Git/CloneCommand.php (hunk 1 above)
Comment (viktorprogger) [reply src/App/Command/Git/CloneCommand.php:15]: WDYT about "Git repository clone command"?

Comment (devanych) [reply src/App/Command/Git/CloneCommand.php]: Replace on "Clone of package repositories" :)

**Comment by devanych** [comment]
> Need add info about command to readme

Done.

**Review by vjik — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Pull request merged**